openhcl/virt_mshv_vtl/src/lib.rs

```diff
@@ -1399,13 +1400,15 @@ impl<'a> UhProtoPartition<'a> {
 
```

Comment:
I can't leave a comment, but remove the TODO TDX comment above this. I'd also update the comment to note that on TDX, we register for exceptions directly without the hypervisor, but on non-TDX platforms we rely on the hypervisor for this intercept.